Integrations/integration-opswat-metadefender-v2.yml

```diff
@@ -0,0 +1,342 @@
+description: multi-scanning engine uses 30+ anti-malware engines to scan files for
+  threats, significantly increasing malware detection.
+configuration:
+- display: Server URL (e.g. http://localhost:8008/metascan_rest/)
+  name: url
+  defaultvalue: http://localhost:8008/metascan_rest/
+  type: 0
+  required: true
+- display: API Key - Needed in cloud based solutions
+  name: api_key
+  defaultvalue: ""
+  type: 4
+  required: false
+- display: Cloud based
+  name: cloud
+  defaultvalue: "false"
+  type: 8
+  required: false
+- display: Trust any certificate (unsecure)
+  name: insecure
+  defaultvalue: "true"
+  type: 8
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: "false"
+  type: 8
+  required: false
+- display: The high threshold
+  name: highPercnt
+  defaultvalue: "66"
+  type: 0
+  required: false
+- display: 'The low threshold '
+  name: lowPercnt
+  defaultvalue: "34"
+  type: 0
+  required: false
+script:
+  script: |
+    import os
+    import requests
+    import json
+    import shutil
+
+
+    # disable insecure warnings
+    requests.packages.urllib3.disable_warnings()
+
+    ''' GLOBAL VARS '''
+    PARAMS = demisto.params()
+    BASE_URL = PARAMS['url'] + '/' if PARAMS['url'][-1] != '/' else PARAMS['url']
+    USE_SSL = not PARAMS.get('insecure', False)
+    API_KEY = PARAMS.get('api_key', '')
+    USE_CLOUD = PARAMS.get('cloud', False)
+    HIGH_THRESHOLD = int(PARAMS.get('highPercnt', 66))
+    LOW_THRESHOLD = int(PARAMS.get('lowPercnt', 34))
+
+
+
+
+    if not demisto.params()['proxy']:
+        del os.environ['HTTP_PROXY']
+        del os.environ['HTTPS_PROXY']
+        del os.environ['http_proxy']
+        del os.environ['https_proxy']
+
+    DEFAULT_HEADERS = {
+        'Accept': 'application/json'
+    }
+
+    HTTP_ERROR_CODES = {
+         '400': 'Request failed, got status 400: bad request. Check command parameters',
+         '401': 'Request failed, got status 401: unauthorized. Check your API Key',
+         '404': 'Request failed, got status 404: not found. Check integration URL Address',
+     }
+
+    ''' HELPER FUNCTIONS '''
+    def http_req(method='GET', url_suffix='', file=None, parse_json=True):
+        files = None
+        url = BASE_URL + url_suffix
+        headers = DEFAULT_HEADERS
+        if USE_CLOUD:
+            headers['apikey'] = API_KEY
+        if file:
+            headers['filename'] = file['file_name']
+            files = {file['file_name']:file['file']}
+        LOG(f'running request with url={url}\n\tfiles={files}')
+        if method.upper() == 'GET':
```

Comment:
do we really need to split this to methods? can't we pass to `GET` request `files=None` ?